File: src/main/java/io/vlingo/schemata/model/sourcing/Result.java
package io.vlingo.schemata.model.sourcing;

import io.vlingo.actors.testkit.TestUntil;

import java.util.ArrayList;
import java.util.List;

/**
 * @author Chandrabhan Kumhar
 * Used to set result in every schemata
 */
public class Result {
    public List<Object> applied = new ArrayList<> ();
    public boolean defined;
    public boolean described;
    public boolean renamed;
    public boolean recategorised;
    public TestUntil until = TestUntil.happenings ( 1 );
}

File: src/main/kotlin/io/vlingo/schemata/model/sourcing/Result.kt
package io.vlingo.schemata.model.sourcing

import io.vlingo.actors.testkit.TestUntil
import java.util.*

class Result {
    var applied: MutableList<String> = ArrayList()
    var tested1: Boolean = false
    var tested2: Boolean = false
    var until = TestUntil.happenings(1)
}
